### DISCUSSION TITLE
Better discord implementation?

### DISCUSSION THREAD

**Discussion by Anymusxd** [ppy/osu#29801] [Ideas]
Right now, when playing a map it just shows the name and "clicking circles"

It would be cool if it showed things like:

-Max pp achievable
-Mods used
-Star rating
-BPM
-Lenght of the map
-(maybe) max combo? (obviously not updating every note, but like every 5-10 seconds when max combo is not yet broken)

**Comment by bdach** [comment]
For the beatmap related info I believe there is already a button to open the map the user is playing in the browser? Not sure the rest is needed.

**Reply by Anymusxd** [reply]
I understand its not "needed" but a cool little addition
